src/wallet/rpcwallet.cpp

```diff
@@ -2406,6 +2406,11 @@ static UniValue getwalletinfo(const JSONRPCRequest& request)
             "  \"paytxfee\": x.xxxx,                (numeric) the transaction fee configuration, set in " + CURRENCY_UNIT + "/kB\n"
             "  \"hdseedid\": \"<hash160>\"            (string, optional) the Hash160 of the HD seed (only present when HD is enabled)\n"
             "  \"private_keys_enabled\": true|false (boolean) false if privatekeys are disabled for this wallet (enforced watch-only wallet)\n"
+            "  \"scanning\":                        (json object) current scanning details, or null if no scan is in progress\n"
+            "    {\n"
+            "      \"duration\" : xxxx              (numeric) elapsed milliseconds since scan start\n"
```

Comment:
This should probably be seconds. I doubt that a user is interested in knowing the exact milliseconds

Reply:
Yeah but I thought `"duration": 1.23` seconds is worst because of the decimal part. Happy to change though.

Reply:
Heh, could truncate

Reply:
Changed to seconds, only integral part.